unity-renderer/Assets/DCLPlugins/ECS7/Systems/InputSenderSystem/ECSInputSenderSystem.cs

```diff
@@ -37,8 +41,13 @@ private static void Update(State state)
                 if (!model.dirty)
                     continue;
 
+                inputResults[i].value.model.lastEntity++;
+                if (inputResults[i].value.model.lastEntity < MIN_ENTITY_TARGET || inputResults[i].value.model.lastEntity >= MAX_ENTITY_TARGET)
+                {
+                    inputResults[i].value.model.lastEntity = MIN_ENTITY_TARGET;
+                }
```

Comment:
You can use `Mathf.Clamp` instead.

Reply:
It's not a Clamp. When the value is out of range, it always sets to the MIN

Reply:
Oh, you are right, disregard this comment. Sorry!